glib2/test/test-regex.rb

```diff
@@ -190,4 +190,8 @@ def test_match_flags
       assert_equal(2, regex.string_number("a_name"))
     end
   end
+
+  def test_regex_escape_string
+    assert_equal(GLib::Regex.escape_string("a.b*c"), "a\\.b\\*c")
```

Comment:
The first argument of `assert_equal` is the expected value because the expected value is more important for readers who read test code:

```ruby
assert_equal("a\\.b\\*c", GLib::Regex.escape_string("a.b*c"))
```